### PR TITLE
Add option to override Pack install timeout

### PR DESCRIPTION
**Pull request by hnanchahal**
Allow overriding the default(600) pack install timeout.

### CODE REVIEW

**Comment by armab** [comment]
There are some linting issues reported by CI to fix, check the log:
https://travis-ci.org/github/StackStorm/st2/jobs/744707184#L2025-L2029

File: CHANGELOG.rst [hunk 1]
```diff
@@ -15,6 +15,11 @@ Changed
 * Improve the st2-self-check script to echo to stderr and exit if it isn't run with a
   ST2_AUTH_TOKEN or ST2_API_KEY environment variable. (improvement) #5068
 
+* Added timeout parameter for packs.install action to help with long running installs that exceed the 
+default timeout of 600 sec which is defined by the python_script action runner (improvement) #5084
+
+Contributed by @hnanchahal
```
Comment (armab) [inline CHANGELOG.rst:21]: Is the linting suggests that the indentation is wrong here.

File: contrib/packs/actions/install.meta.yaml [hunk 2]
```diff
@@ -37,3 +37,9 @@
       description: "Set to True to skip pack dependency installations."
       required: false
       default: false
+    timeout:
+      default: 600
```
Comment (armab) [inline contrib/packs/actions/install.meta.yaml:41]: What was the default timeout before this PR?

Comment (hnanchahal) [reply contrib/packs/actions/install.meta.yaml:41]: the default was 600 only.

Comment (armab) [reply contrib/packs/actions/install.meta.yaml:41]: So the default was 600, now it's 600? No change?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -15,6 +15,11 @@ Changed
 * Improve the st2-self-check script to echo to stderr and exit if it isn't run with a
   ST2_AUTH_TOKEN or ST2_API_KEY environment variable. (improvement) #5068
 
+* Added timeout parameter for packs.install action to help with long running installs that exceed the 
+  default timeout of 600 sec which is defined by the python_script action runner (improvement) #5084
+
+Contributed by @hnanchahal
```
Comment (armab) [inline CHANGELOG.rst:21]: This line is still needs indentation adjustment
```suggestion
  Contributed by @hnanchahal
```

**Review by armab — APPROVED** [review]
LGTM.

Thanks for the contribution.

File: contrib/packs/actions/install.meta.yaml (hunk 2 above)
Comment (blag) [reply contrib/packs/actions/install.meta.yaml:41]: Users should have been able to override this parameter before. I don't see the what we gain by making it explicit if we're just going to keep the same default value. @hnanchahal ?

**Pull request merged**